index.html

```diff
@@ -0,0 +1,218 @@
+            </p>
+            <p>
+              the personality section of this persona was based off the Myers
+              Briggs personality test. According to the Myers Briggs, there are
+              16 potential user personality types. Our template integrates
+              questions from this online personality test example into a series
+              of sliding bar graphics. If you&apos;re confused as to what the
+              sliders mean, check out the Myers Briggs basics article.
+            </p>
+          </div>
+        </section>
+        <section class="sketches">
+          <h2 class="subtitle">Initial Sketches</h2>
+          <picture>
+            <source
+              srcset="src/images/sketches_dekspot.jpg"
+              media="(min-width: 1440px)"
+            />
+            <source
+              srcset="src/images/sketches_tablet.jpg"
+              media="(min-width: 768px)"
+            />
+            <img src="src/images/sketches_mobile.jpg" alt="picture of" />
+          </picture>
+          <div class="text">
+            <p>In the beginning, It&apos;s just about brainstorming.</p>
+            <p>
+              I list out a jumble of words and draw tiny sketches related to
+              what I&apos;m working on (and find the parallels later).
+              Personally, I prefer paper with a grid so I can convey things
+              responsively. Most of the time perfection isn&apos;t an issue and
+              the sketches are just for me.
+            </p>
+          </div>
+        </section>
+        <section class="final-design">
+          <h2 class="subtitle">Final Design</h2>
+          <picture>
+            <source
+              srcset="src/images/final-design_dekstop.jpg"
+              media="(min-width: 1440px)"
+            />
+            <source
+              srcset="src/images/final-design_tablet.jpg"
+              media="(min-width: 768px)"
+            />
+            <img
+              src="src/images/final-design_mobile.jpg"
+              alt="picture of street lamp"
+            />
+          </picture>
+          <div class="text">
+            I played with adding a wireframe of the website flow that would lead
+            you in to each screen. For each of those screens I added a floating
+            footer that allows users to see where they are in that particular
+            flow and jump around as needed. We&apos;re digging this
+            InVision-made wireframe.
+          </div>
+        </section>
+      </div>
+    </main>
+    <footer class="footer">
+      <div class="wrapper footer__wrapper">
+        <div>
+          <p class="footer_text">Need a project?</p>
+          <a class="footer__link" href="tel:+4733378901">(239) 555-0108</a>
+          <a class="footer__link" href="mailto:name@rapidtables.com">
+            tanya.hill@example.com
+          </a>
+        </div>
+        <div class="social--media">
```

Comment:
WCAG 2.1 requires each <a> anchor link to have a text describing the purpose of the link using either plain text or an <img> with the alt attribute set.